Source/Core/Duality/Launcher/LauncherArgs.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Linq;
+
+namespace Duality.Launcher
+{
+	public class LauncherArgs
```

Comment:
As per best practices, please do XML docs for all public members - bonus points for the private ones 🙂